components/zcash_protocol/Cargo.toml

```diff
@@ -35,6 +35,9 @@ incrementalmerkletree = { workspace = true, optional = true }
 [dev-dependencies]
 proptest.workspace = true
 
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = ['cfg(zcash_unstable, values("zfuture"))'] }
+
```

Comment:
Once the workspace-level lints section is in place, AFAICT this should be unnecessary (because this crate is in the workspace):
```suggestion
```